modules/ROOT/edits/bind-api-configure-service-mesh-CRD.adoc

```diff
@@ -0,0 +1,124 @@
+
+* Ready: The binding is now ready to be used. 
+* Failed: The binding was not created successfully. An error occurred. 
+* Binding: The binding is in the process of being associated with specific application.
+
+In the following example, a binding called `customers` is created to bind the `anypoint-service-mesh-adapter` adapter with the `customers` service. This binding is created in the `nto-payments` namespace, which is the same namespace where the adapter was previously created, and is associated with the API that has the ID as 1234.
+
+[source,text,linenums]
+----
+NAMESPACE      NAME        ADAPTER                         API ID  SERVICE NAME    STATUS
+nto-payments   customers   anypoint-service-mesh-adapter   1234    customers       Ready
+----
+
+== Methods for Binding an API
+
+You can bind an API to a service using Anypoint Platform or Anypoint Service Mesh, based on your preference:
+
+* <<bind_the_service, Bind an API created using Anypoint Platform>>: If you created your API using Anypoint Platform, use this procedure.
+* <<bind_using_ASM, Bind an API created using Anypoint Service Mesh>> : If you created your API using Anypoint Service Mesh, use this procedure.
+
+
+[[bind_the_service]]
+== Bind an API Created Using Anypoint Platform
+
+Before you bind the service, ensure that you complete the task prerequisite.
+
+*Task Prerequisite*
+From the API Administration Tab of API Manager, obtain the API ID to create the binding using this method.
+
+To bind the service with an API, perform the following steps:
+
+. Go to *Anypoint Platform > API administration* and expand the API version name to display the instances.
+. Click an instance name and copy the API ID that is displayed. 
+Ensure that you save this API ID to a text editor.
++
+image::api-id.png[]
++
+. To retrieve the service name from Kubernetes, with which you want to bind the API, run the command:
++
+`kubectl get svc -n <namespace>`
+. Verify that your microservice port is named following the requirements requested by Istio: xref:https://istio.io/docs/setup/additional-setup/requirements/[].
+. If you created your API using Anypoint Platform, use `spec.parameters.apiId`:
++
+[source,text,linenums]
+----
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceBinding
+metadata:
+  name: <binding name> # example: orders-api-binding
+  namespace: <namespace name> # example: default
+  labels: 
+    service-mesh.mulesoft.com/kind: adapter-binding
+spec:
+  instanceRef:
+    name: <adapter name> # example: bookinfo-service-mesh-adapter
+  parameters:
+    apiId: <api ID> # example: 15722113
+    service: <service name> # example: orders-service
+----
++
+The binding is activated within a few seconds and the API is successfully registered to the proxy.
```

Comment:
Do we need to say that the steps were successful by including this text?
I would omit it.

Reply:
I'm keeping it as it reinforces the users and tells them what they should be seeing on their screen. Done! Thanks!